lib/DownloadUtils.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Created by AsherS on 8/20/15.
+ */
+
+var fs = require('fs');
```

Comment:
1. Please move utils to a 'utils' folder
2. I'm pretty sure qHttp.request.read does exactly what you're looking for here [https://www.npmjs.com/package/q-http]